homework/index.js

```diff
@@ -31,16 +33,60 @@
   }
 
   function main(url) {
-    fetchJSON(url, (err, repositories) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-        return;
-      }
-      createAndAppend('pre', root, { text: JSON.stringify(repositories, null, 2) });
-    });
-  }
+    fetch(url)
+      .then(data => {
+        data.json()
```

Comment:
There seems to be a problem here, when you're using an arrow function with `{}` you don't have implicit returns. So change this to either of:
```javascript
.then(data => data.json())
// or
.then(data => {
  return data.json();
})
```